### PR TITLE
Add custom_alias.py

### PR DESCRIPTION
**Pull request by mrvanes**
This feature adds a static custom alias endpoint to SVS. It needs the configuration update in the inacademia-dev repo https://github.com/inacademia-development/inacademia-dev/pull/10

### CODE REVIEW

File: src/svs/custom_alias.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        :param config: The SATOSA proxy config
+        """
+        super().__init__(*args, **kwargs)
+        # self.config = config
+        if 'locations' in config:
+            self.locations = config['locations']
+
+    def register_endpoints(self):
+        url_map = []
+        for endpoint, alias in self.locations.items():
```
Comment (tziai) [inline src/svs/custom_alias.py]: self.locations might not be defined

File: src/svs/custom_alias.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            endpoint = endpoint.strip("/")
+            logger.info("{} registering {} - {}".format(self.logprefix, endpoint, alias))
+            url_map.append(["^%s/" % endpoint, self._handle])
+        return url_map
+
+    def _handle(self, context):
+        path = context._path
+        endpoint = path.split("/")[0]
+        target = path[len(endpoint) + 1:]
+        alias = "%s/%s" % (self.locations[endpoint], target)
```
Comment (tziai) [inline src/svs/custom_alias.py]: Dito

Comment (mrvanes) [reply src/svs/custom_alias.py]: It's impossible to reach that line without a valid key for locations[endpoint] but I included it in the try, just to be sure.

File: src/svs/custom_alias.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    logprefix = "CUSTOM_ALIAS_SERVICE:"
+
+    def __init__(self, config, *args, **kwargs):
+        """
+        :type config: satosa.satosa_config.SATOSAConfig
+        :param config: The SATOSA proxy config
+        """
+        super().__init__(*args, **kwargs)
+        # self.config = config
+        if 'locations' in config:
```
Comment (tziai) [inline src/svs/custom_alias.py]: You should remove the if statement

Comment (mrvanes) [reply src/svs/custom_alias.py]: Good catch

**Comment by tziai** [comment]
It now looks okay to me.

**Pull request merged**